themes/default/content/docs/intro/pulumi-service/access-tokens.md

```diff
@@ -0,0 +1,135 @@
+1. Select **Create token**.
+
+An Organization Access Token must have a name that is unique among all Organization Access Tokens assigned to it. This allows tokens taking operations on behalf of your organization to be identifiable in the event that one is compromised. This name must be unique even among deleted tokens, in order to maintain the integrity of Audit Log Events which persist the token’s name (even for a deleted token) as part of the event (see below). Any other organization admin can delete this token; it is not owned by the admin which created it.
+
+Creation of any Organization Access Token, and the user who performed it, is logged as an Audit Log Event.
+
+### Viewing Organization Access Tokens
+
+Organization Access Tokens are viewed by going to the organization’s Settings page, then selecting Access Tokens from the menu. As with Personal Access Tokens, this table displays all tokens belonging to your organization, and at what time they were last successfully used to carry out an operation. Only organization admins of an organization with an Enterprise Growth or Business Critical subscription will see access to this page.
+
+### Deleting an Organization Access Token
+
+An Organization Access Token can be deleted by any organization admin at any time. No other members are able to delete organization tokens.
+
+From the organization's homepage, follow the same steps as for a Personal Access Token:
+
+1. Select **Settings**.
+1. Navigate to **Access Tokens**.
+1. Select the ellipsis button.
+1. Choose **Delete token**. You will be prompted in a dialog to confirm your choice.
+
+If you choose to delete a token, its access will immediately be revoked and all further operations using it will fail as unauthorized.
+
+### Auditing Organization Token Actions
+
+Since an organization can have multiple Organization Access Tokens, it’s necessary to be able to identify them uniquely in Audit Log Events. All Audit Log Events which were triggered by an Organization Access Token will surface the token’s unique name, and in the event of Audit Log Export, the token’s UUID as well.
+
+### Permissions/Authorization
+
+Put simply: Organization Access Tokens behave like an organization member with [stack write permissions](https://www.pulumi.com/docs/intro/pulumi-service/projects-and-stacks/#stack-permissions) to all of the organization’s stacks. They do not grant any privileges to view the console, or to create additional Organization or Personal Access Tokens. See below for a full list of accessible APIs:
+
+#### API Access
+
+See the Pulumi [Service REST API docs](https://www.pulumi.com/docs/reference/service-rest-api/) for more information about each API endpoint.
+
+| Action | Org Token Access |
+|--------|------|
+| Stacks |
+|--------|------|
+| List Stacks | ✅ |
+| Get Stack | ✅ |
+| Get Stack State | ✅ |
+| Transfer Stack |  |
+| Delete Stack |  |
```

Comment:
Is it true that org tokens cannot delete a stack? 

If so that seems like a bug. What is the rationale?

Reply:
Same with deleting a team.

Reply:
Updating to include that we support this, and will correct this in the main payload. Docs fixed in [22f33ac](https://github.com/pulumi/pulumi-hugo/pull/1429/commits/22f33ac8dc2afa81aedcfeeb294d1ffeae486f23)

Reply:
Not true, but has been corrected in docs [22f33ac](https://github.com/pulumi/pulumi-hugo/pull/1429/commits/22f33ac8dc2afa81aedcfeeb294d1ffeae486f23)